backend/codesirius/problems/views/execution_constraint.py

```
@@ -0,0 +1,51 @@
+                many=True,
+                context={"problem": problem},
+                partial=True,
+            )
+            if serializer.is_valid(raise_exception=True):
+                serializer.save()
+                return CodesiriusAPIResponse(
+                    status_code=status.HTTP_207_MULTI_STATUS,
+                    data=serializer.data,
+                )
+        except Problem.DoesNotExist:
+            raise ValidationError("Problem not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)